expat/doc/xmlwf.xml

```diff
@@ -383,19 +383,35 @@ supports both.
 	<command>&dhpackage;</command> prints a single line describing
 	the problem to standard output.  If a file is well formed,
 	<command>&dhpackage;</command> outputs nothing.
-	Note that the result code is <emphasis>not</emphasis> set.
 	</para>
   </refsect1>
+
+  <refsect1>
+  <title>EXIT STATUS</title>
+    <para>The following exit status codes are returned:
+    <variablelist>
+      <varlistentry>
+        <term><option>0</option></term>
+        <listitem><para>The input files are well-formed.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>1</option></term>
+        <listitem><para>An internal error occurred.</para>
```

Comment:
I found this case as well:
https://github.com/libexpat/libexpat/blob/83e1688c74ef4d9034e6b47a6595dee480be6147/expat/xmlwf/xmlwf.c#L1057-L1061
Maybe it could be classified as an output error? Other ideas?

Reply:
Nice catch.  It would make sense to me to change that case to exit(3) and document it appropriately, since it quite different to the result=1 and 2 cases.  WDYT?  I can update the PR to do this.

Reply:
Good idea! :+1: